File: SRM.Domain/Data/HoatDongData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SRM.Domain.Data
{
    public class HoatDongData : ModelBase
    {
        public Guid? LoaiHoatDongId { get; set; }
        public string? TenLoaiHoatDong { get; set; }
        public string? Ten { get; set; }
        public string? NoiDung { get; set; }
        public string? GhiChu { get; set; }
        public string? DiaChi { get; set; }
        public decimal? KinhPhi { get; set; }
        public int? SoTrang { get; set; }
        public int? SoTiet { get; set; }
        public string? FileDinhKem { get; set; }
        public DateTime? NgayBatDau { get; set; }
        public DateTime? NgayKetThuc { get; set; }
        public string? DuongDan { get; set; }
        public string? ChuNhiem { get; set; }
        public string? ThanhVienThamGias { get; set; }
        public string? PhanChiaSuDongGop { get; set; }
    }
}

File: srm.client/src/models/HoatDongData.ts
import { ModelBase } from './ModelBase';
import { Dayjs } from 'dayjs';

export interface HoatDongData extends ModelBase {
  loaiHoatDongId?: string;
  tenLoaiHoatDong?: string;
  ten?: string;
  noiDung?: string;
  ghiChu?: string;
  diaChi?: string;
  kinhPhi?: number;
  soTrang?: number;
  soTiet?: number;
  fileDinhKem?: string;
  ngayBatDau?: string | Dayjs;
  ngayKetThuc?: string | Dayjs;
  duongDan?: string;
  chuNhiem?: string;
  thanhVienThamGias?: string;
  phanChiaSuDongGop?: string;
}